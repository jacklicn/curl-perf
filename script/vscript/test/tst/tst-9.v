

sub next(v)
    return v+1
end


var v = next(1)
print( v )

print( next(1) )

print( next( next(1) ) )


sub print_both( a, b )
  print(a)
  print(b)
end

print_both( next(1), next( next(1) ) )

