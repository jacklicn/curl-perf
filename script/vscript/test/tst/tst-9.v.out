*** syntax ok ***
*** semantics and code generation ok ***

>2

>2

>3

>2

>3
